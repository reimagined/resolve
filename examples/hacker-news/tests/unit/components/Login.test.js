import React from 'react'
import { shallow } from 'enzyme'

import Login from '../../../client/components/Login'

it('renders correctly', () => {
  const markup = shallow(<Login location={{ search: '' }} />)

  expect(markup).toMatchSnapshot()
})
