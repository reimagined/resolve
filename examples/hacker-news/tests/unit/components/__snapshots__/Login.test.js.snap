// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Login
    location={
      Object {
        "search": "",
      }
    }
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <AuthForm
          action="/login"
          buttonText="login"
          title="Login"
        />,
        <AuthForm
          action="/register"
          buttonText="create account"
          title="Create account"
        />,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "action": "/login",
          "buttonText": "login",
          "title": "Login",
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "action": "/register",
          "buttonText": "create account",
          "title": "Create account",
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <AuthForm
            action="/login"
            buttonText="login"
            title="Login"
          />,
          <AuthForm
            action="/register"
            buttonText="create account"
            title="Create account"
          />,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "action": "/login",
            "buttonText": "login",
            "title": "Login",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "action": "/register",
            "buttonText": "create account",
            "title": "Create account",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
